### DISCUSSION TITLE
Windows Exe Download

### DISCUSSION THREAD

**Discussion by marianoka** [Security-Onion-Solutions/securityonion#4966] [General] [answered]
Hi all!
I just need to find the way to download and exe like doug did at the blog. Any help, welcome.
Thanks
Mariano
![imagen](https://user-images.githubusercontent.com/9213379/127323316-4e7d2077-4605-4ecd-a22d-2ffa52ee1062.png)

**Comment by dougburks** [comment, marked as answer]
Go to Alerts or Hunt and find a log for an EXE. Click on the alert to access the Quick Actions Menu. Then click the PCAP action.

For more information, please see:
https://docs.securityonion.net/en/2.3/alerts.html#actions
https://docs.securityonion.net/en/2.3/hunt.html#actions
https://docs.securityonion.net/en/2.3/pcap.html
